rabbitmq/datadog_checks/rabbitmq/rabbitmq.py

```diff
@@ -534,7 +534,7 @@ def _get_metrics(self, data, object_type, custom_tags):
             for path in keys[:-1]:
                 root = root.get(path, {})
 
-            value = root.get(keys[-1], None)
+            value = root.get(keys[-1], None) if isinstance(root, dict) else None
```

Comment:
```suggestion
            value = root.get(keys[-1], None) if isinstance(root, dict) else None  # In 3.1.x queue_totals is a list instead of a dict
```